.github/workflows/pull_request.yml

```diff
@@ -32,10 +34,23 @@ jobs:
           args: -d -z
 
       - uses: actions/upload-artifact@v4
+        id: upload-artifact
         with:
           name: WeakAuras-PR#${{ github.event.number }}-${{ github.sha }}
           path: .release/
 
+      - uses: exercism/pr-commenter-action@v1.5.1
+        if: ${{ contains(github.event.pull_request.labels.*.name, '🆕 Feature Preview') }}
```

Comment:
relying on 🆕 being part of the string sounds like a time bomb

Reply:
Hmm, how do you reckon? It's just a unicode codepoint (that either I or @Stanzilla wrote into the label's name a million years ago). We could drop the emojis from the name, certainly, but (because the expression syntax is fairly limited) trying to preserve them & also not write emojis into our source files would involve IMO worse hackery:

```yml
      - uses: exercism/pr-commenter-action@v1.5.1
      	if: ${{ contains(join(github.event.pull_request.labels.*.name, ', '), 'Feature Preveiew') }}
```

Reply:
Ah okay, then don't bother.